script/Arcueid/Day 9/09_00_ARC09_1.txt

```diff
@@ -171,135 +171,135 @@ The adults are coming.
 [91cd303842aee015a4de94f74fab642e552ce73e]{
 -- Page 6, Offset 21326.
 -- この惨状に気がついて、血相を変えてやってくる。
--- TRANSLATION HERE
+Realizing the terrible spectacle in front of them, their faces quickly turn to anger.
 }
 [8471e2ae173e01d897de6d5191381839f230f365]{
 -- Page 7, Offset 21327.
 -- 「なんて事だ―――――」
--- TRANSLATION HERE
+"What have you done―――――"
 }
 [1c6d9f3b3eaa01cbba7a3e64348889cb145634b4]{
 -- Page 7, Offset 21328.
 -- おとなたちは秋葉を連れていく。
--- TRANSLATION HERE
+The adults take Akiha away.
 }
 [5086f40c391d81fe3661eefe542843186d375cb7]{
 -- Page 7, Offset 21329.
 -- 倒れた<子|モ><供|ノ>は死んだまま。
--- TRANSLATION HERE
+The collapsed child remains dead.
 }
 [326c4cb14a6c4e3ffca70dda4d4017f737ca9499]{
 -- Page 7, Offset 21330.
 -- 遠くのそらには、白い白い入道雲。
--- TRANSLATION HERE
+The bright, white clouds float in the skies far above.
 }
 [f20fc8a12d5d5c8631109dd9f6ca156ee064be10]{
 -- Page 7, Offset 21331.
 -- ボクはひとり残されて、ぼんやりと夏の空を見上げている。
--- TRANSLATION HERE
+Being the only one left behind, I vacantly look up toward the summer sky.
 }
 [335621dd001afd868d96a59b1015bbbcf68f01fa]{
 -- Page 8, Offset 21332.
 -- 「おまえが殺したのか――――」
--- TRANSLATION HERE
+"Were you the one that killed him――――"
 }
 [a551722d637091c92c5e1cb8610e6d147eb59e28]{
 -- Page 8, Offset 21333.
 -- おとなたちは叫んでいる。
--- TRANSLATION HERE
+The adults are screaming at me.
 }
 [6b5a92eed0f57d8ff2a3f4e4d9a799efc843ea49]{
 -- Page 8, Offset 21334.
 -- 子供をころしたボクの名前を叫んでいる。
--- TRANSLATION HERE
+They are screaming the name of the one who killed the child.
 }
 [23749a287cf4ae1fb70d95d0d64a253c0837a60d]{
 -- Page 8, Offset 21335.
 -- たった二文字の言葉を、気がちがったように叫んでいる。
--- TRANSLATION HERE
+The word is only two syllables long, but they scream it like they were losing their mind.
 }
 [1a0c4ddc9dfe2ca6f2a2357808ccbf2b36b2d6a8]{
 -- Page 8, Offset 21336.
 -- たった二文字。
--- TRANSLATION HERE
+All over two syllables.
 }
 [62614e07ed7576217839cc5924ddbc4dbb3dcd92]{
 -- Page 8, Offset 21337.
 -- おとなたちはそろって、両手を真っ赤に染めたボクを、シキと呼んでおさえつけた。
--- TRANSLATION HERE
+The adults gather up and pin down on the one they call Shiki, whose hands are soaked in a deep red.
```

Comment:
this TL is missing boku and could do with some reflowing imo
>おとなたちはそろって、両手を真っ赤に染めたボクを、シキと呼んでおさえつけた。
The adults and gather up and pin me—the one they call Shiki with hands stained a deep red—to the ground.